package.json

```diff
@@ -36,6 +36,7 @@
         "cross-spawn": "^7.0.3",
         "glob": "^8.0.3",
         "globby": "^11.0.0",
+        "proxyquire": "^2.1.3",
```

Comment:
Do you have a 3PP for this? Also, it doesn't look like this is used in production code, so should it be in `devDependencies` instead of `dependencies`?

Reply:
My bad! I intended this to a devDependencies as we discussed in the tea time last week. I have moved it now.